index.rst

```diff
@@ -6,24 +6,119 @@
+
+
+| import lsst.ts.cRIOpy.M1M3FATable as M1M3FATable
+| FATABLE = M1M3FATable.FATABLE
+|
+| Actuator ID (as in Figure 1): 227
+| ID to index:  M1M3FATable.actuatorIDToIndex(227) : 61
+| index to ID: FATABLE[index][M1M3FATable.FATABLE_ID]: 227
+| Secondary index: FATABLE[index][M1M3FATable.FATABLE_SINDEX]: 46
+| Y index: FATABLE[index][M1M3FATable.FATABLE_YINDEX]: 42
+| Z index: FATABLE[index][M1M3FATable.FATABLE_ZINDEX]: 61
+|
+| Since all actuators provide force in the Z direction, the ZINDEX and the INDEX are equal.
+|
+| The following actuator is one of the few that provides force in the X direction.
+|
+| Actuator ID (as in Figure 1): 135
+| ID to index:  M1M3FATable.actuatorIDToIndex(135) : 34
+| X index: FATABLE[index][M1M3FATable.FATABLE_XINDEX]: 2
+
+
+Table1: Navigating between actuator IDs and sequential indices.
+
+Bump Test States
+==================================
+
+As the bump test cycles through the testing, the EFD location:
+
+| lsst.sal.MTM1M3.logevent_forceActuatorBumpTestStatus
+|
+
+returns where in the bump test cycle the test is.  Figure 5 shows the progression of these states.  At the conclusion of the test for each actuator, the state will either be 6 (PASSED) or 7 (FAILED).
+
+.. image:: ./_static/Bump_Test_States.png
+
+Figure 5. This shows the bump test states as logged in  lsst.sal.MTM1M3.logevent_forceActuatorBumpTestStatus.
+
+XYZ forces vs Cylinder forces
+==============================
+
+The force data in the EFD includes both the forces applied to the primary and secondary cylinders, as well as the forces in the X, Y, and Z directions.  These forces are not the same, as will be explained in this section.  Referring to Figure 6, we see that the primary cyclinder applies force in the axial direction, which is along the optical axis of the telescope, and is referred to as the Z direction.  So the force applied by the primary cylinder is equal to the Z-force.  The secondary cylinder however, is at a 45 degree angle to the Z-axis.  So if we want to apply a force in the lateral direction (X or Y), we need to apply a force with the secondary cylinder, and then a negative force with the primary cylinder so that the resultant is in the lateral direction.
+
+.. image:: ./_static/Force_Schematic.png
+
+Figure 6. This schematic of the actuators shows the relation between the cylinder forces and the resulting axial and lateral forces.
+
+The primary and secondary forces are what is actually measured, and the X, Y, and Z forces are calculated from these forces and the known geometry.  So Figures 3 and 4 show the Y and Z forces.  Figures 7 and 8 show the target and actual forces in the primary and secondary cylinders.  Note the negative primary force being applied during the secondary bump test.
```

Comment:
Add something on the tune:

As the angle of the secondary cylinder is 45 degrees, transforming mirror coordinate systems forces into secondary cylinder is (mirror_force) / sqrt(2). Assume Z and Y DAA should produce Z force +10 N, and Y +25 N. Then the primary cylinder force would be 10 - (25 / srtq(2)), and secondary cylinder force would be 25 / sqrt(2)